tests/e2e/Makefile

```diff
@@ -3,7 +3,9 @@ include $(shell find tests/e2e -name "Makefile" ! -wholename "tests/e2e/Makefile
 export VERTX_IMG ?= jaegertracing/vertx-create-span:operator-e2e-tests
 export ELASTIC_IMG ?= docker.elastic.co/elasticsearch/elasticsearch-oss:6.8.6
 export OPERATOR_IMAGE_NEXT ?= ${IMG_PREFIX}/jaeger-operator:next
-export ASSERT_IMG ?= ${IMG_PREFIX}/asserts-e2e:$(shell date +%s)
+# get "tests/assert-jobs" directry commit id and use that as build tag to avoid frequent build and push container image into a remote registry
+export ASSERT_IMG ?= ${IMG_PREFIX}/asserts-e2e:$(shell git log -n 1 --pretty=format:%h ./tests/assert-jobs/)
```

Comment:
This change should not be needed. The image name was stored in a file that avoids triggering the image build. This mechanism should be used again

Reply:
in downstream, I do not want to build the image on each run. the name stored in a file only valid for that run.

Reply:
In that case, something that can be done in the future is to provide a way to get the image from a registry if its build

Reply:
@iblancasa you mean something like, build if not exist?

Reply:
Yes. But it something we can improve in the future